haproxy-ingress/templates/_podtemplate.yaml

```diff
@@ -208,7 +228,6 @@ spec:
 {{- end }}
 
 {{- define "haproxy-ingress.controller.ports" -}}
-ports:
 {{- if .Values.controller.enableStaticPorts }}
   - name: http
```

Comment:
Something here is leaving an empty line in the template, couldn't see a way to fix it. `helm template ... | grep '^[ ]*$' out` and you can see it. Not a big deal btw, it's not breaking the template.

Reply:
It's because the indentation was done with `nindent`, which inserts an extra newline before the template text.

Reply:
Argh yes, I reviewed all the templates when migrating from helm repo and updated to v0.8+, fixed a lot of them. Completely ignored it. Thanks for the fix.